### PR TITLE
Add namespace to Module message

### PR DESCRIPTION
**Pull request by zaporter-work**
@ethanlook Mentioned that it would be easier for the frontend if we just provided the public namespace (if it exists) rather than making them parse the module id (or perform another lookup). Now that the namespace is attached to the module on the backend, this should be pretty easy to add on the backend.

### CODE REVIEW

File: proto/viam/app/v1/app.proto [hunk 1]
```diff
@@ -877,6 +877,9 @@ message Module {
   string module_id = 1;
   // The id of the organization that owns the module
   string organization_id = 10;
+  // The public namespace of the organization that owns the module
+  // This is empty if no public namespace is set
+  string public_namespace = 12;
```
Comment (RoxyFarhad) [inline proto/viam/app/v1/app.proto:882]: nit: maybe add this so the numbering is correctly in order ?

**Review by RoxyFarhad — APPROVED** [review]
:)

File: proto/viam/app/v1/app.proto (hunk 1 above)
Comment (zaporter-work) [reply proto/viam/app/v1/app.proto:882]: Add this so the numbering is in the correct order? There is already an `=11;` 

Do you just mean sort the entries in the message by the id?

Comment (RoxyFarhad) [reply proto/viam/app/v1/app.proto:882]: yeah just sorted in ID so its clear what the next value needs to be

**Review by michaellee1019 — APPROVED** [review]

File: proto/viam/app/v1/app.proto (hunk 1 above)
Comment (zaporter-work) [reply proto/viam/app/v1/app.proto:882]: Sure -- I just left because it was already unsorted but happy to re-sort

**Review by ethanlook — APPROVED** [review]

**Pull request merged**